src/components/notification/channel/googlechat.ts

```diff
@@ -0,0 +1,233 @@
+ * Permission is hereby granted, free of charge, to any person obtaining a copy   *
+ * of this software and associated documentation files (the "Software"), to deal  *
+ * in the Software without restriction, including without limitation the rights   *
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      *
+ * copies of the Software, and to permit persons to whom the Software is          *
+ * furnished to do so, subject to the following conditions:                       *
+ *                                                                                *
+ * The above copyright notice and this permission notice shall be included in all *
+ * copies or substantial portions of the Software.                                *
+ *                                                                                *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     *
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       *
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    *
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         *
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  *
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  *
+ * SOFTWARE.                                                                      *
+ **********************************************************************************/
+import axios from 'axios'
+
+import { NotificationMessage } from '../../../interfaces/notification'
+import { GoogleChatData } from '../../../interfaces/data'
+import { log } from '../../../utils/pino'
+
+export const sendGoogleChat = async (
+  data: GoogleChatData,
+  message: NotificationMessage
+) => {
+  const notifType =
+    message.meta.type[0].toUpperCase() + message.meta.type.substring(1)
+  let chatMessage
+
+  switch (message.meta.type) {
+    case 'start':
+      chatMessage = {
+        cards: [
+          {
+            header: {
+              title: 'Monika Notification',
+              subtitle: `New ${notifType} from Monika`,
+              imageUrl: 'https://bit.ly/3kckaGO',
+            },
+            sections: [
+              {
+                widgets: [
+                  {
+                    textParagraph: {
+                      text: `<b>Message</b> New monika monitoring started`,
+                    },
+                  },
+                  {
+                    textParagraph: {
+                      text: `<b>Time</b> ${message.meta.time}`,
+                    },
+                  },
+                  {
+                    textParagraph: {
+                      text: `<b>From</b> ${message.body}`,
+                    },
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      }
+      break
+    case 'termination':
+    case 'incident':
```

Comment:
For both incidents and recoveries, because Google Chat doesn't render the sections with indents like Microsoft Teams:
![image](https://user-images.githubusercontent.com/16670475/140696927-619417a4-9d2a-42a2-8c4f-45ce634a2bf6.png)

In the google chat:
![image](https://user-images.githubusercontent.com/16670475/140696833-4d32d9f3-1dc5-49ef-83b6-4b77853e0ce2.png)

I think it's best to add a colon after the field name such as:
```
Message: ${message.summary}
URL: ${message.meta.url} 
```
and so on.

Or another option, give a line break:
```
Message:
${message.summary}

URL:
${message.meta.url}
```
and so on.

Reply:
![image](https://user-images.githubusercontent.com/964106/140702454-6773b9c8-a317-4700-9b93-8bfbb3db8036.png)
adding colons

Reply:
![image](https://user-images.githubusercontent.com/964106/140705697-1fa7a0b7-3e2f-44ff-a626-3dc55d08a220.png)